### ISSUE TITLE
Support for providing URL for MP3 file?

### ISSUE THREAD

**Issue by giggo** [denniedegroot/com.google.tts#11]
Hi
do you plan to provide support to generate a MP3 file and generate a URL to this file so that it can be played back for example at a Sonos device?
Ciao,
Gagga

**Comment by denniedegroot** [comment]
Not right now, but maybe in the future.

**Comment by JPe2** [comment]
Would be very appreciated if you add the function a mentioned by @giggo, reading the forum, lots of Homey owners looking for a way to play sounds or text thru Sonos.

**Comment by denniedegroot** [comment]
@JPe2 Sonos is not a chromecast device so this will never work with Sonos.

**Comment by JPe2** [comment]
O.K. Just the mentioned Enhancement would be fine, any chance you can do that this year?

**Comment by giggo** [comment]
No. Bit generatiing a MP3 and streaming the URL to the MP3 would work. 

Gagga
